src/trio_util/_async_value.py

```diff
@@ -1,6 +1,10 @@
 # pylint: disable=line-too-long,multiple-statements
 from contextlib import contextmanager
-from typing import TypeVar, Generic, AsyncIterator, Tuple, ContextManager, Callable, overload
+from typing import TypeVar, Generic, AsyncIterator, Tuple, Callable, overload
+try:
+    from typing import ContextManager
+except ImportError:
+    from contextlib import AbstractContextManager as ContextManager  # pylint: disable=ungrouped-imports
```

Comment:
this is the first user-visible change in a few years, so it needs a new package release

Reply:
it's pushed to v0.8.0

I also invited you as pypi package owner